modules/install/pages/upgrade.adoc

```diff
@@ -150,11 +150,11 @@ Note, however, that _Graceful Failover_ for Data Service nodes, with _Delta Reco
 [#node-certificate-validation]
 == Node Certificate Validation
 
-In Couchbase Enterprise Server Version 7.2+, the node-name _must_ be correctly identified in the node certificate as a Subject Alternative Name.
+In Couchbase Enterprise Server Version 7.2 or later, the node-name _must_ be correctly identified in the node certificate as a Subject Alternative Name.
 If such identification is not correctly configured, failure may occur when uploading the certificate, or when attempting to add or join the node to a cluster.
 
-When upgrade is performed from a pre-7.2 cluster to a 7.2+ cluster, if a new, 7.2 node is introduced to the cluster without the node-name being correctly configured in its certificate, a warning is provided.
-Likewise, if a new certificate is uploaded onto a 7.2 node, a warnng is provided.
+When upgrade is performed from a pre-7.2 cluster to a 7.2+ cluster, if a new, 7.2+ node is introduced to the cluster without the node-name being correctly configured in its certificate, a warning is provided.
+Likewise, if a new certificate is uploaded onto a 7.2+ node, a warning is provided.
 In neither case does failure occur.
```

Comment:
This is not correct. A failure can occur.

Reply:
Actually, the upgrade passages above are outdated now - basically, the latest rewrite just points to the certificates overview so that the main info is in just one place.